### ISSUE TITLE
Command Line Option "zeroDowntimeUpdate" is not working

### ISSUE THREAD

**Issue by fbessai** [Axway-API-Management-Plus/apim-cli#370]
### APIM-CLI version

1.13.5

### API-Management version

7.7.0.20220228

### Bug description

Using  option "zeroDowntimeUpdate" with command line has no effect.

### Steps to reproduce

- In the API Manager, Have a published API, with at least one subscription.
- Make a change to the api-definition/swagger file  (or any modification for the sake of testing)
- Update the API with apim-cli, uszing option " -zeroDowntimeUpdate  "  (with no argument as defined in the help section) 

### Relevant log output

```shell
- The API is updated but the api-accesses are revoked => all subscriptions are removed


Suggestion :
In this class : https://github.com/Axway-API-Management-Plus/apim-cli/blob/435feec0178eee4b4f4eaf6e7f40ca6781b89e62/modules/apis/src/main/java/com/axway/apim/apiimport/lib/cli/CLIAPIImportOptions.java , 

 it looks like the "zeroDowntimeUpdate" is declared with no arguments (Line 95 ). But it is read as if it has arguments (Line 143)
```

**Comment by rathnapandi** [comment]
@fbessai, 

You are right, will fix it in the next release. 
Currently you can set up the ZDD variable via env.properties file. 

# You can set the switch zeroDowntimeUpdate to true to update APIs with zero downtime as before. 
# That means that when updating a published API, the CLI always creates a completely new API, 
# configures it according to the old API and finally clients are switched to the new API. 
# This new API then has a new ID and runtime quotas are lost. The configured quotas are 
# transferred to the new API. 
# zeroDowntimeUpdate=true

**Comment by rathnapandi** [comment]
@fbessai, 

You are right, will fix it in next release. 
Currently you can set up the ZDD variable via env.properties file. 
```bash
# You can set the switch zeroDowntimeUpdate to true to update APIs with zero downtime as before. 
# That means that when updating a published API, the CLI always creates a completely new API, 
# configures it according to the old API and finally clients are switched to the new API. 
# This new API then has a new ID and runtime quotas are lost. The configured quotas are 
# transferred to the new API. 
zeroDowntimeUpdate=true
```

**Comment by fbessai** [comment]
@rathnapandi , thanks for the alternative solution

**Comment by rathnapandi** [comment]
Fixed as part of 1.13.6
